src/AWS.Deploy.CLI/Commands/GenerateDeploymentProjectCommand.cs

```diff
@@ -230,8 +230,14 @@ private async Task GenerateDeploymentRecipeSnapShot(Recommendation recommendatio
             recipe.Name = recipeName;
             recipe.CdkProjectTemplateId = null;
             recipe.CdkProjectTemplate = null;
+            recipe.PersistedDeploymentProject = true;
```

Comment:
For now so that persisted deployment projects show up top in the recommendations lets set `RecipePriority` to `1000`. Call this a constant like `DEFAULT_PERSISTED_RECIPE_PRIORITY = 1000`.

Reply:
Added.